readme.md

```diff
@@ -198,6 +215,7 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
 * **Question:** Describe the purpose of both morgan and body-parser. Why do you think morgan is only being run when the application is run in development mode?
 
 * **Your Answer:**
+morgan is used for logging request details. The body-parser reads and stores the input from the body.  We do not use the body-parser on the devlopment server because if we are saving all the requests, then the server would come back slow.
```

Comment:
We need body-parser on all versions of the server else we would not have access to req.body
morgan is dev specific since it it not required for a live server.